bundles/org.openhab.automation.pythonscripting/src/main/resources/lib/openhab/helper.py

```diff
@@ -10,10 +11,11 @@
 
 # **** REGISTER LOGGING AND EXCEPTION HOOK AS SOON AS POSSIBLE ****
 Java_LogFactory = java.type("org.slf4j.LoggerFactory")
-LOG_PREFIX = "org.openhab.core.automation.pythonscripting"
+LOG_PREFIX = "org.openhab.automation.pythonscripting"
 if '__file__' in scope:
     file_package = os.path.basename(scope['__file__'])[:-3]
     LOG_PREFIX = "{}.{}".format(LOG_PREFIX, file_package)
+file_package = os.path.basename(scope['__file__'])[:-3]
```

Comment:
I think this is not needed

Reply:
Thanks. I thought I pulled your latest and this was there.